webClient/src/app/uploader-panel/uploader-panel.component.ts

```diff
@@ -246,13 +248,16 @@ export class UploaderPanelComponent {
             error => {},
             () => {
               console.log('Finished with', file.name);
+              this._snackbar.open('Upload Successful!', 'Dismiss', {
```

Comment:
Great opportunity to embed the Zowe Notification manager here too! Example of it being used here: https://github.com/zowe/zlux-file-explorer/blob/97615c0bb272fcafeb2f319506f67b46d7fcbad3/src/app/components/filebrowseruss/filebrowseruss.component.ts#L738